app/models/user.rb

```diff
@@ -7,6 +7,10 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: :author_id
   has_many :likes, foreign_key: :author_id
 
+  def admin?
+    role == 'admin'
```

Comment:
- [ ] This definition is not recognized. When I try to navigate to any Post routes I get this error. This is because you have not added the "role" attribute to your Users table. You need to run a migration and add a "role" attribute to your users schema as it is necessary for checking who an admin is and to avoid these errors.

  Error when I visit any post routes | User schema does not have "role" attribute
  :---:|:---:
  ![ad](https://user-images.githubusercontent.com/81468675/186656752-71272c1e-5a79-490d-9847-7e5f4528c62b.jpg) | ![user-role](https://user-images.githubusercontent.com/81468675/186655746-3dfa005d-907f-45bd-b6c5-73ae481c1e3c.jpg)

  **Note:** After resolving that, on the browser ensure that apart from the post author, the user who is an admin can also delete a post and a comment